test/letters/reducers/index.unit.spec.js

```diff
@@ -38,71 +65,74 @@ describe('letters reducer', () => {
   it('should handle failure to fetch letters', () => {
     const state = lettersReducer.letters(
```

Comment:
Given that this is used so many times, I wonder if it would make sense to abstract this out into something like a `reduce(action) { ...call the letters reducer with a pre-defined initialState and action passed in as arg, return new state...}` that can just manually be called in each test, so: `const newState = reduce(action); expect(newState).to.deep-eql({ oldState with changes });`.

I mean, this is already written so probably not worth it at this point. I'm just genuinely curious what you think about that approach for testing reducers in general.

Reply:
🤔 
Yeah...yeah, that's not bad. Then if we needed to change the initial state to test a thing, we can just call it with an optional parameter if we were so inclined. I like it.